tools/remove_project_hold.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Allow authorized users to remove a hold on a project.
+
+$relPath="./../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'Project.inc'); // validate_projectID() Project() remove_holds()
+include_once($relPath.'metarefresh.inc'); // metarefresh()
+
+require_login();
+
+$projectid = validate_projectID('projectid', @$_POST['projectid']);
+
+$project = new Project($projectid);
+if (!$project->can_be_managed_by_current_user)
+{
+    die(_('You are not authorized to manage this project.'));
+}
+
+$return_uri = urldecode($_POST['return_uri']);
+$states = array($_POST['curr_state']);
+
+$project->remove_holds($states);
+$title = _("Removing hold");
+metarefresh(1, $return_uri, $title, $title);
```

Comment:
One last thought - we should immediately redirect the user and not have a 1 second delay. This will be a better user experience. So lets change the 1 second to 0 seconds.